search_service/models/index_map.py

```diff
@@ -0,0 +1,83 @@
+import textwrap
+
+DEFAULT_INDEX_MAP = textwrap.dedent("""
```

Comment:
is this the same as the one defined in databuilder es_publisher?

Reply:
it is!